broadcast/StandardJsonInput/README.md

```diff
@@ -0,0 +1,24 @@
+# Standard JSON input
+
+This folder contains the standard JSON input to the Solidity compiler for the contracts in this repository.
+
+Many block explorer (e.g., Etherscan-based ones) support verifying contract code directly through their interface by using these files.
+
+They can be used as a last resort if other verification scripts don't work.
+
+## Warning
+
+Because of a [deployment issue](https://github.com/cowprotocol/composable-cow/issues/93), the contract bytecode doesn't fully match the compiled version.
+This means that the standard input for the following contracts may not lead to full verification:
+
+ - ExtensibleFallbackHandler
+ - ComposableCoW
+ - TWAP
+ - GoodAfterTime
+ - PerpetualStableSwap
+ - TradeAboveThreshold
+ - StopLoss
+ - CurrentBlockTimestampFactory
```

Comment:
Maybe is best to not include a complete list, and instead leave instructions on how to add new contracts